### PR TITLE
[CIS-1725] Message delivery status docs

### PR DESCRIPTION
**Pull request by evsaev**
### 🔗 Issue Link

CIS-1725

⚠️ The PR is targeting `feature/message-delivery-status` branch ⚠️

### 🎯 Goal

Write the docs about message delivery status:
- an overview
- a guide how to show user avatars who's seen the message

### 📝 Changes

- `ChatMessageDeliveryStatusView` is made a `UIControl` so it's easy to handle events and see visual feedback when an interaction happens
- 2 pages are added to iOS section in docs 

### 🎨 UI Changes

https://user-images.githubusercontent.com/12818985/162444083-0f9510c3-50b5-4545-9a38-c30161c7c8e5.mov

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] This change follows zero ⚠️ policy (required)
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

**Review by evsaev — COMMENTED** [review]
Ouch, images in the customization guide are **1mb** each 😬 How do you compress images @GetStream/ios-developers?

UPD: Fixed ✅

File: Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift [hunk 1]
```diff
@@ -482,6 +482,19 @@ open class ChatMessageListVC:
                 "Tapped an avatarView. To customize the behavior, override messageContentViewDidTapOnAvatarView. Path: \(indexPath)"
             )
     }
+    
+    /// This method is triggered when delivery status indicator on the message at the given index path is tapped.
+    /// - Parameter indexPath: The index path of the message cell.
+    open func messageContentViewDidTapOnDeliveryStatusIndicator(_ indexPath: IndexPath?) {
+        guard let indexPath = indexPath else { return log.error("IndexPath is not available") }
```
Comment (bielikb) [inline Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift:489]: @evsaev is this action going to be piped to one of the delegates?

Comment (evsaev) [reply Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift:489]: This method is triggered when the delivery indicator is tapped and can be overridden in `ChatMessageListVC` subclass if needed. The same is done for all other `ChatMessageContentViewDelegate`'s methods

**Review by bielikb — APPROVED** [review]
Left one comment/question.

Nice docs!

**Pull request merged**